docs/superpowers/plans/2026-04-29-self-ref-chain-fixture-test.md

```
@@ -0,0 +1,320 @@
+      }
+  ```
+
+- [ ] **Step 2: Run the new test to verify it passes**
+
+  ```bash
+  cargo test -p raven self_ref_variable_not_in_scope_across_deep_source_chain 2>&1 | tail -20
+  ```
+
+  Expected output includes:
+
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)